bom/openhab-addons/pom.xml

```diff
@@ -231,11 +231,13 @@
       <artifactId>org.openhab.binding.digiplex</artifactId>
       <version>${project.version}</version>
     </dependency>
+<!--
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.digitalstrom</artifactId>
       <version>${project.version}</version>
     </dependency>
+-->
```

Comment:
Why is this getting commented out?

Reply:
On macOS Catalina 10.15.7 the full build is reporting error by these components. Shall I revert back?

Reply:
changes are reverted